### PR TITLE
request

### PR DESCRIPTION
**Pull request by sushi-ae**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/thirsty.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "very thirsty.",
+  "repo": "https://github.com/frapdotbmp/thirsty.github.io",
+  "owner": {
+    "username": "thirsty",
```
Comment (WilliamDavidHarrison) [inline domains/thirsty.json:5]:
```suggestion
    "username": "frapdotbmp",
```

File: domains/thirsty.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    "email": "frapdotpng@gmail.com"
+  },
+  "record": {
+    "CNAME": "thirsty.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/thirsty.json:9]:
```suggestion
    "CNAME": "frapdotbmp.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please make a website

**Comment by WilliamDavidHarrison** [comment]
No website + no response in nearly a month.

**Pull request closed without merging**